core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -71,6 +71,7 @@ object Defaults {
   val QueuedMaxRequestBytes = -1
   val RegistrationHeartbeatIntervalMs = 2000
   val RegistrationLeaseTimeoutMs = 18000
+  val MetadataLogCheckIntervalMs = 10000
```

Comment:
I think the value passed in before was 10ms rather than 10000 ms.  That being said, I wonder if this configuration parameter is specific to the `LocalLogManager` class and if it deserves its own config value.  Does it appear in a KIP anywhere?  I didn't see it in KIP 631.

Reply:
Yea, good point about the config. I am not sure if this should even be a user configurable variable. I'll revert to using a constant for now w/ 10ms for the `LocalLogManager`

Reply:
And good catch w/ the 10 ➝ 10000 🤦